samples-generator/lib/createBuildingsTile.js

```diff
@@ -161,13 +195,20 @@ function generateBatchTableBinary(buildings) {
 
     var batchTableJson = {
         cartographic : {
+            name : 'CartographicBuffer',
             byteOffset : 0,
-            componentType : 'DOUBLE',
-            type : 'VEC3'
+            byteLength : cartographicBuffer.length,
+            componentType : 0x1406, // TODO: Logical error, not sure what to do, 'DOUBLE' was here originally but GLTF only supports Floats
+            type : 'VEC3',
+            count : buildingsLength
         },
+
         code : {
+            name: 'Code',
             byteOffset : cartographicBuffer.length,
-            componentType : 'UNSIGNED_BYTE',
+            count: codeBuffer.length,
+            byteLength: codeBuffer.length,
+            componentType : 0x1401,
             type : 'SCALAR'
         }
```

Comment:
These changes might go against the "legacy" batch table binary schema. `byteLength` and `count` are not valid properties in https://github.com/CesiumGS/3d-tiles/blob/master/specification/schema/batchTable.schema.json#L19-L40.

Reply:
added a legacy mode when !glb and !gltf that uses the old behavior selectively :+1: